keen/tests/client_tests.py

```diff
@@ -8,20 +8,37 @@
 import keen
 from keen.client import KeenClient
 from keen.tests.base_test_case import BaseTestCase
+from mock import patch, MagicMock
 import sys
 
 __author__ = 'dkador'
 
 
+class MockedRequest(object):
+    def __init__(self, status_code, json_response):
+        self.status_code = status_code
+        self.json_response = json_response
+
+    def json(self):
+        return {"result": self.json_response}
+
+
+class MockedFailedRequest(MockedRequest):
+    def json(self):
+        return self.json_response
+
+
+@patch("requests.Session.post")
 class ClientTests(BaseTestCase):
     def setUp(self):
         super(ClientTests, self).setUp()
+        self.api_key = "2e79c6ec1d0145be8891bf668599c79a"
```

Comment:
since `self.api_key` is no longer used anywhere else, I think you can just change this here (and on lines 38 + 39) to `api_key`